<?xml version="1.0" ?>
<launch>


  <group ns="ds1">
    <param name="input_topic_name" value="/ur5/ur5_cartesian_velocity_controller_sim/ee_state_pose"/>
    <param name="output_topic_name" value="/ds_desired_velocity"/>
    <param name="output_filtered_topic_name" value="/ds1/filter/desired_velocity"/>
    <rosparam command="load" file="$(find ds_motion_generator)/config/cpr_4tasks/cpr_reach_right.yml"/>
    <node name="ds_node" pkg="ds_motion_generator" type="active_motionGenerator_node" output="screen"/>

    <!-- # doesn't work yet, but I should follow this style -->
    <!-- <remap from="/ds1/filter/desired_velocity" to="/lwr/joint_controllers/passive_ds_command_vel" /> -->

  </group>

  <!--group ns="DS_2">
  	<param name="input_topic_name" value="/real_position"/>
  	<param name="output_topic_name" value="/ds2/desired_velocity"/>
  	<param name="output_filtered_topic_name" value="/ds2/filter/desired_velocity"/>
	<rosparam command="load" file="$(find ds_motion_generator)/config/DS1_params.yaml"/>
	<node name="DS_2" pkg="ds_motion_generator" type="active_motionGenerator_node" output="screen"/>
  </group-->


  <!--group ns="DS_2">
  	<param name="gmm_path" value="$(find ds_motion_generator)/data/DS2"/>
  	<param name="max_des_vel" value=".3"/>
  	<node name="motionGenerator" pkg="ds_motion_generator" type="active_motionGenerator_node" output="screen"/>
  </group-->

    <node name="ee_splitter" pkg="admittance_control" type="cpr_ee_state_splitter_node" output="screen"/>



</launch>
